java-frontend/src/test/java/org/sonar/java/cfg/CFGTest.java

```diff
@@ -937,6 +950,65 @@ public void while_loop_with_continue() {
       terminator(Tree.Kind.CONTINUE_STATEMENT, 3));
     cfgChecker.check(cfg);
   }
+  @Test
+  public void continue_in_try_finally() {
+    final CFG cfg = buildCFG("void fun() { while (foo()) {\n" +
+      "      try {\n" +
+      "        bar(\"try\");\n" +
+      "        continue;\n" +
+      "      } finally {\n" +
+      "        qix(\"finally\");\n" +
+      "      }\n" +
+      "    }}");
+    final CFGChecker cfgChecker = checker(
+      block(
+        element(IDENTIFIER, "foo"),
+        element(METHOD_INVOCATION)
+      ).terminator(WHILE_STATEMENT).successors(0, 4),
+      block(element(TRY_STATEMENT)).successors(3),
+      block(
+        element(Tree.Kind.IDENTIFIER, "bar"),
+        element(STRING_LITERAL, "try"),
+        element(METHOD_INVOCATION)
+      ).successors(2).exceptions(1),
+      new BlockChecker(CONTINUE_STATEMENT, 1).hasNoExitBlock(),
```

Comment:
I would prefer having `terminator(CONTINUE_STATEMENT, 1).hasNoExitBlock()` here instead, in order to stay closer from the syntax used in other tests.